content/docs/ops/runbook/restoring-rds.md

```diff
@@ -0,0 +1,80 @@
+---
+menu:
+  docs:
+    parent: runbook
+layout: ops
+
+title: Restoring RDS
+---
+
+## Introduction
+Many of the major services for the cloud.gov platform are backed by Postgres databases provided by RDS.  In the case of outages or corruption, it is imperative an operator be able to quickly find the affected database and restore it from a snapshot if possible.  These instructions are augment the documenation from the official RDS documentation.
```

Comment:
"These instructions are augment" should maybe be "These instructions augment". And "documenation" should be "documentation".